docker/build.sh

```diff
@@ -1,18 +1,92 @@
 #!/bin/bash
 
-# Single source of truth for building Sam.
-# @ Jackie Roberti
-# @ Doug Voet
-#
-# Provide command line options to do one or several things:
-#   jar : build sam jar
-#   -d | --docker : provide arg either "build" or "push", to build and push docker image
-# Jenkins build job should run with all options, for example,
-#   ./docker/build.sh jar publish -d push
+HELP_TEXT="$(cat <<EOF
+ Build jar and docker images.
+   jar: build jar
+   -d | --docker : (default: no action) provide either "build" or "push" to
+           build or push a docker image.  "push" will also perform build.
+   -g | --gcr-registry: If this flag is set, will push to the specified GCR repository.
+   -k | --service-account-key-file: (optional) path to a service account key json
+           file. If set, the script will call "gcloud auth activate-service-account".
+           Otherwise, the script will not authenticate with gcloud.
+   -h | --help: print help text.
+ Examples:
+   Jenkins build job should run with all options, for example,
```

Comment:
Can you clarify this?  The example given does not include all options.

Reply:
yep